ui/package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "ng": "ng",
     "prebuild": "rimraf ./dist",
+    "build": "ng build --prod --output-hashing=none && node build-script.js",
```

Comment:
If you remove the `--prod` flag, we get a much better idea of what's in the resulting bundle. Maybe we leave this and add a dev version? I was thinking this could be named like `build:wc` or something of the sort

Reply:
Good point, I'll update this now.